README.md

```diff
@@ -1,6 +1,17 @@
 # project-build-examples
 Sample projects to demonstrate project CI/CD features of the [`project-build-plugin`](http://axonivy.github.io/project-build-plugin/snapshot).
 
+## Versions and Branches
+You are currently on the master branch. Features shown here might still be under development and may not yet be released!
+
+If you are working with a LTS or LE version consider to switch to the corresponding branch:
+
+|Axon Ivy Version                         | Branch          |
+|-----------------------------------------|-----------------|
+|Latest LTS version 8.0                   | [8.0](tree/8.0) |
+|Latest LE version 9.3                    | [9.3](tree/9.3) |
+|Next LE version 9.4 (under development)  | [master](#)     |
```

Comment:
I'd rather go for 'Current LE version' ... there should not be a need to adjust readme files after releasing.

Reply:
also in later becoming "9.4" branch .. should not be any need for adjustment. So we should rather point out as you do, that we maintain multiple branches for each and every version. But not refering specific ones. Maybe 'master' and 'lts' branches as exception